src/ui/pages/GameUI.jsx

```diff
@@ -1,11 +1,12 @@
 import { css } from '@emotion/react';
 import { useLayoutEffect, useMemo, useRef, useEffect } from 'react';
 import { useParams, Navigate, Link } from 'react-router-dom';
+// @ts-ignore
 import { _ } from 'lodash';
-import { levels } from './LevelSelectPage/LevelSelectPage';
```

Comment:
Jak teraz nie wykorzystujemy `levels` to można to wyrzucić z exportu w `LevelSelectPage`.